netlify.toml

```diff
@@ -17,3 +17,7 @@
   from = "/docs/*"
   to = "https://trinodb.github.io/docs.trino.io/:splat"
   status = 200
+
+[[redirects]]
+  from = "/star"
+  to = "https://github.com/trinodb/trino/blob/master/.github/star.png"
```

Comment:
Add empty line at end of file

Reply:
done

Reply:
It doesn't show up in the diff or online editor, but when I go to my branch and edit via online editor it shows the newline at the end.